docs/index.md

```diff
@@ -0,0 +1,167 @@
+
+!!! info ""
+    **Looking for a quick run through of the core utilities?**
+
+  	Check out [this detailed blog post](https://aws.amazon.com/blogs/opensource/simplifying-serverless-best-practices-with-lambda-powertools/) with a practical example.
+
+## Install
+
+Powertools is available in PyPi. You can use your favourite dependency management tool to install it
+
+* [poetry](https://python-poetry.org/): `poetry add aws-lambda-powertools`
+* [pip](https://pip.pypa.io/en/latest/index.html): `pip install aws-lambda-powertools`
+
+**Quick hello world example using SAM CLI**
+
+```bash
+sam init --location https://github.com/aws-samples/cookiecutter-aws-sam-python
+```
+
+### Lambda Layer
+
+Powertools is also available as a Lambda Layer. It is distributed via the [AWS Serverless Application Repository (SAR)](https://docs.aws.amazon.com/serverlessrepo/latest/devguide/what-is-serverlessrepo.html).
+We have two layers available, one with core dependencies `aws-lambda-powertools-python-layer` and one with extras `aws-lambda-powertools-python-layer-extras` such as `pydantic` which is required for the parser.
+
+!!! note
+    Extras layer does not support Python 3.6 runtime. This layer is also includes all extra dependencies and is 22.4MB zipped, ~155MB unzipped.
+
+| App | ARN
+|----------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------
+| [aws-lambda-powertools-python-layer](https://serverlessrepo.aws.amazon.com/applications/eu-west-1/057560766410/aws-lambda-powertools-python-layer) | arn:aws:serverlessrepo:eu-west-1:057560766410:applications/aws-lambda-powertools-python-layer
+| [aws-lambda-powertools-python-layer-extras](https://serverlessrepo.aws.amazon.com/applications/eu-west-1/057560766410/aws-lambda-powertools-python-layer-extras) | arn:aws:serverlessrepo:eu-west-1:057560766410:applications/aws-lambda-powertools-python-layer-extras
+
+If using SAM, you can include this SAR App as part of your shared Layers stack, and lock to a specific semantic version. Once deployed, it'll be available across the account this is deployed to.
+
+=== "template.yml"
+
+    ``` yaml
+      AwsLambdaPowertoolsPythonLayer:
+        Type: AWS::Serverless::Application
+        Properties:
+          Location:
+            ApplicationId: arn:aws:serverlessrepo:eu-west-1:057560766410:applications/aws-lambda-powertools-python-layer
+            SemanticVersion: 1.9.0 # change to latest semantic version available in SAR
+    ```
+
+
+This will add a nested app stack with an output parameter `LayerVersionArn`, that you can reference inside your Lambda function definition:
+
+```yaml
+  Layers:
+    - !GetAtt AwsLambdaPowertoolsPythonLayer.Outputs.LayerVersionArn
+```
+
+
+Here is the list of IAM permissions that you need to add to your deployment IAM role to use the layer:
+
+=== "template.yml"
+
+    ```yaml
```

Comment:
perhaps for completion (can be done later, don't bother if you're already got a lot on your plate)... add an IAM Role or Policy resource with this policy in it.

Reply:
Added an IAM role that will be assumed by CloudFormation.